scheduler/starter/modules.go

```diff
@@ -0,0 +1,208 @@
+package starter
+
+import (
+	"fmt"
+	"net"
+	"os"
+	"time"
+
+	"github.com/apache/thrift/lib/go/thrift"
+	log "github.com/sirupsen/logrus"
+	"golang.org/x/net/proxy"
+
+	"github.com/twitter/scoot/cloud/cluster"
+	"github.com/twitter/scoot/common"
+	"github.com/twitter/scoot/common/dialer"
+	"github.com/twitter/scoot/runner"
+	"github.com/twitter/scoot/runner/runners"
+	"github.com/twitter/scoot/saga"
+	"github.com/twitter/scoot/saga/sagalogs"
+	"github.com/twitter/scoot/scheduler/scheduler/config"
+	"github.com/twitter/scoot/scheduler/server"
+	"github.com/twitter/scoot/workerapi/client"
+)
+
+const DEFAULT_SOCKS_PORT = "50001"
```

Comment:
Does all the low level proxy code belong here, vs common?

Reply:
imo common should only contain functionality that is used by > 1 service.